packages/react-dropdown/src/index.js

```diff
@@ -74,7 +75,8 @@ const Dropdown = ({
   children,
   name = 'dropdown',
   twoColumn = true,
-  selectedItems = [],
+  initialSelectedItems = [],
+  selectedItems,
```

Comment:
we need a check that throws a `console.error` if the user tries to set both props at the same time, like it happens with React's own `input` elements